Custom Domain content updates

tamigoodall opened the pull request:
Custom Domain content updates
Review links:
https://docs-content-staging-pr-8492.herokuapp.com/docs/custom-domains
https://docs-content-staging-pr-8492.herokuapp.com/docs/custom-domains/auth0-managed-certificates
https://docs-content-staging-pr-8492.herokuapp.com/docs/custom-domains/self-managed-certificates
https://docs-content-staging-pr-8492.herokuapp.com/docs/custom-domains/additional-configuration
https://docs-content-staging-pr-8492.herokuapp.com/docs/custom-domains/troubleshoot-custom-domains
https://docs-content-staging-pr-8492.herokuapp.com/docs/custom-domains/set-up-cloudfront

<!---
Notes:
- Your PR should conform to our [Contributing Guidelines](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md)
- If applicable, add details to the [update feed](https://github.com/auth0/docs/tree/master/updates)
- Make sure your PR gets deployed to a Heroku [Review App](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#review-apps) without errors
- Please include a short description
- If your PR is a work in progress, title it [DO NOT MERGE]
--->

tamigoodall committed:
Custom Domain content updates

Custom Domain content updates

tamigoodall committed:
Typos

Typos

tamigoodall committed:
Update set-up-cloudfront.md

typo

tamigoodall committed:
Sidebar

Added tables to index.md
Updated sidebar entries

tamigoodall committed:
Added reverse proxy info and other stuff

Added reverse proxy parameters and values
Created an include for additional feature setup
Added note about only one custom domain per tenant
Cleaned up text
Added Keep reading links
Reordered steps on certificate docs

tamigoodall committed:
Update self-managed-certificates.md

Added warning about CNAME requirement

tamigoodall committed:
Added links to tenant settings

Added links to tenant settings in various files
Cleaned up text

tamigoodall committed:
Moved TXT and CNAME stuff

Moved TXT and CNAME stuff

tamigoodall committed:
Fixed sample domain names

Fixed sample domain names

tamigoodall committed:
Update self-managed-certificates.md

Fixed broken image

tamigoodall committed:
Update self-managed-certificates.md

Typo

erjohnson commented on articles/custom-domains/troubleshoot-custom-domains.md:
```diff
@@ -0,0 +1,64 @@
+  - configure-auth0-managed-certificates
+---
+
+# Troubleshooting Custom Domains
```
should the filename just be custom-domains/troubleshoot?

tamigoodall replied:
yeah - probably - I'll change it

tamigoodall committed:
CNAME flattening note and links

CNAME flattening note
Misc link updates in rest of content

tamigoodall committed:
Updated intros

Updated intros based on Jose's training materials

joseeantonior commented on articles/custom-domains/self-managed-certificates.md:
```diff
@@ -1,91 +1,130 @@
+To set up your custom domain using self-managed certificates, you need to provide your domain name to Auth0, verify that you own that domain, configure the reverse proxy, and then complete additional configuration for the Auth0 features you are using.
 
-You'll need to register and own the domain name to which you're mapping your Auth0 domain.
+<%= include('./_provide-domain-name') %>
```
This is including some generic steps in order to add the domain name to Auth0. However, these generic steps (both the screenshot and the written instructions) are based on the `Auth0-Managed Certificates` option. So, on the self-managed certificates page, it's showing options to add with the a0-managed instructions.

tamigoodall replied:
Fixed

erjohnson commented on config/sidebar.yml:
```diff
@@ -641,8 +641,22 @@ articles:
             url: /custom-domains/self-managed-certificates
+            children:
+
+              - title: Reverse Proxy Example
```
Should probably link to custom-domains/set-up-azure-cdn + custom-domains/set-up-cloudfront

tamigoodall replied:
Got it

tamigoodall committed:
Added conditional screenshot include

Added conditional screenshot include for self- and Auth0-managed files

tamigoodall committed:
Final edits

Changed name of troubleshooting file and links
Added Azure CDN reverse proxy file links

tamigoodall committed:
Update self-managed-certificates.md

Added Cloudflare link

joseeantonior commented on articles/custom-domains/auth0-managed-certificates.md:
```diff
@@ -0,0 +1,66 @@
+
+1. Go to [Dashboard > Tenant Settings](${manage_url}/#/tenant), add the CNAME verification record listed in the Dashboard to your domain's DNS record.
+
+  ![](/media/articles/custom-domains/auth0-managed.png)
```
Last review is for this line (forgot to hit a button before)

tamigoodall committed:
Update set-up-cloudfront.md

Removed toc

tamigoodall committed:
Merge branch 'master' into okr-custom-domains-improvements

tamigoodall committed:
Reverse Proxy doc fixes

Rerverse Proxy doc fixes

tamigoodall committed:
Added image titles

Added image titles

tamigoodall committed:
Subscription text update

Subscription text update

tamigoodall committed:
Update index.md

Updated Certificate management text

tamigoodall committed:
Update additional-configuration.md

fixed broken link

donflopez reviewed (approved):
🚀 🌔

machuga commented on articles/custom-domains/_provide-domain-name.md:
```diff
@@ -0,0 +1,21 @@
+
+2. Select the **Custom Domains** tab.
+
+  <% if (platform === "auth0") { %>
```
If a customer has multiple domains enabled, what happens in this situation, @tamigoodall ?

tamigoodall replied:
You receive an error message that states that you have reached the maximum number of custom domains allowed for one tenant.

machuga replied:
We allow it in special cases and are looking at adding it in 2020. I can send you a tenant with a second domain if you need one for testing.

tamigoodall replied:
They receive an error message.

machuga replied:
Is this going to be a problem? How can we support their use case? They are usually large enterprise customers
